packages/x-tailwindcss/src/x-tailwind-plugin/theme.ts

```diff
@@ -88,37 +88,41 @@ export default {
   },
   letterSpacing: {
     sm: '-1%',
-    md: 0,
+    md: '0',
     lg: '2%'
   },
   lineHeight: {
-    sm: 1,
-    md: 1.5,
-    lg: 2
+    sm: '1.2',
+    md: '1.5',
+    lg: '2'
   },
   screens: {
     tablet: '800px',
     desktop: '1200px',
     large: '2800px'
   },
   spacing: {
-    'spacing-1': 1,
-    'spacing-2': 2,
-    'spacing-4': 4,
-    'spacing-8': 8,
-    'spacing-12': 12,
-    'spacing-16': 16,
-    'spacing-20': 20,
-    'spacing-24': 24,
-    'spacing-32': 32,
-    'spacing-40': 40,
-    'spacing-48': 48,
-    'spacing-56': 56,
-    'spacing-64': 64,
-    'spacing-80': 80,
-    'spacing-96': 96,
-    'spacing-128': 128,
-    'spacing-152': 152
+    1: 1,
+    2: 2,
+    4: 4,
+    8: 8,
+    12: 12,
+    16: 16,
+    20: 20,
+    24: 24,
+    32: 32,
+    40: 40,
+    48: 48,
+    56: 56,
+    64: 64,
+    80: 80,
+    96: 96,
+    128: 128,
+    152: 152,
+    184: 184,
+    216: 216,
+    280: 280,
+    344: 344
```

Comment:
Tailwind defaults `px` if there aren't any (Removing the `px` from the `fontSize` section has the same effect) so it's not mandatory.

I would say we should add it because that way when calling `theme` in `old-ds-plugin/plugin.ts` we won't have to concat the `px` there. It would be useful also in the future because `theme` is heavily used when creating custom components and utilities, so we would have to concat `px` there too when using `spacing`.

Reply:
Is that documented somewhere?

Reply:
fixed, added unit to spacing tokens